docs/tutorials/infrastructure/binary/index.md

```diff
@@ -0,0 +1,431 @@
+sudo chmod +x /usr/bin/temporal-ui-server
+```
+
+At this point, you've downloaded everything you need. The last thing to do is create a `temporal` user on your server that has the appropriate permissions to run the Temporal Service, and directory accessible to this user to store your data in. Run the following commands:
+
+```bash
+sudo useradd temporal
+sudo mkdir /etc/temporal
+sudo chown temporal /etc/temporal
+```
+
+Next, you'll create configuration files for both the Temporal Server and the UI Server in the `/etc` directory.
+
+## Configuring the Temporal Binaries
+
+In this tutorial, you'll configure a connection to a SQLite database, since it doesn't require any additional dependencies. Using your favorite text editor, open a new file called `/etc/temporal-server.yaml`:
+
+```bash
+sudo vim /etc/temporal-server.yaml
+```
+
+Paste the following contents into the file for a starting configuration. You can update any of these values later.
+
+```
+log:
+  stdout: true
+  level: info
+
+persistence:
+  defaultStore: sqlite-default
+  visibilityStore: sqlite-visibility
+  numHistoryShards: 1
+  datastores:
+    sqlite-default:
+      sql:
+        pluginName: "sqlite"
+        databaseName: "/etc/temporal/default.db"
+        connectAddr: "localhost"
+        connectProtocol: "tcp"
+        connectAttributes:
+          cache: "private"
+          setup: true
+
+    sqlite-visibility:
+      sql:
+        pluginName: "sqlite"
+        databaseName: "/etc/temporal/visibility.db"
+        connectAddr: "localhost"
+        connectProtocol: "tcp"
+        connectAttributes:
+          cache: "private"
+          setup: true
+
+global:
+  membership:
+    maxJoinDuration: 30s
+    broadcastAddress: "127.0.0.1"
+  pprof:
+    port: 7936
+
+services:
+  frontend:
+    rpc:
+      grpcPort: 7233
+      membershipPort: 6933
+      bindOnLocalHost: true
+      httpPort: 7243
+
+  matching:
+    rpc:
+      grpcPort: 7235
+      membershipPort: 6935
+      bindOnLocalHost: true
+
+  history:
+    rpc:
+      grpcPort: 7234
+      membershipPort: 6934
+      bindOnLocalHost: true
+
+  worker:
+    rpc:
+      grpcPort: 7239
+      membershipPort: 6939
+      bindOnLocalHost: true
+
+clusterMetadata:
+  enableGlobalNamespace: false
+  failoverVersionIncrement: 10
+  masterClusterName: "active"
+  currentClusterName: "active"
+  clusterInformation:
+    active:
+      enabled: true
+      initialFailoverVersion: 1
+      rpcName: "frontend"
+      rpcAddress: "localhost:7233"
+      httpAddress: "localhost:7243"
+
+dcRedirectionPolicy:
+  policy: "noop"
+```
+
+Save and close the file. Next, you'll create the configuration file for the UI Server. Using your favorite text editor, open a new file called `/etc/temporal-ui-server.yaml`:
+
+```bash
+sudo vim /etc/temporal-ui-server.yaml
+```
+
+Paste the following contents into the file.
+
+```
+temporalGrpcAddress: 127.0.0.1:7233
+port: 8233
+enableUi: true
+cors:
+  allowOrigins:
+    - http://localhost:8233
```

Comment:
This needs to be whatever their custom domain is rather than localhost, otherwise things like Terminate etc won't work as they will fail CORS checks.